src/legacy/core_plugins/data/public/search/search_bar/components/saved_query_management/save_query_form.tsx

```diff
@@ -98,6 +98,12 @@ export const SaveQueryForm: FunctionComponent<Props> = ({
       defaultMessage: 'Title conflicts with an existing saved query',
     }
   );
+  const titleMissingErrorText = i18n.translate(
+    'data.search.searchBar.savedQueryForm.titleMissingText',
+    {
+      defaultMessage: 'Title is required',
```

Comment:
This should say "Name" instead of "Title" since we call it name elsewhere in the form

Reply:
We use 'Title' in the `titleConflictErrorText` and was working towards consistency. If one changes, the other should change too.

Reply:
Good call, using title in the other messages was a mistake I think.